package.json

```diff
@@ -43,6 +43,7 @@
         "@ckeditor/ckeditor5-media-embed": "^39.0.2",
         "@ckeditor/ckeditor5-table": "^39.0.2",
         "@ckeditor/ckeditor5-theme-lark": "^39.0.2",
+        "@fortawesome/fontawesome-free": "^6.7.2",
```

Comment:
If we add the font files, is this necessary?

Reply:
I don't know - didn't try that. I just assumed that it would be better to add via package manager. This is a 100% free package, so we shouldn't run into the issues that we are currently having.

Reply:
Okay I'm thinking the font files are redundant then. I agree it's better to do this with a package manager, but I'm not sure the font files are doing anything if this package handles it.

Reply:
@iccowan Sorry, I'm not following you. This PR didn't add any font files. Do we have them somewhere in the repo now? I agree - that would be unnecessarily duplicative.

Reply:
@kjporter In the diff I see all of these files 
<img width="300" alt="image" src="https://github.com/user-attachments/assets/e1484d06-17c5-4e71-a7b8-9e6a9517347d" />

Reply:
The font files are installed by the package - one in the same